### DISCUSSION TITLE
Add Alpine image for arm64/v8

### DISCUSSION THREAD

**Discussion by ngosang** [dani-garcia/vaultwarden#1449] [Ideas]
[Feature request] Add Alpine image for arm64/v8

You are already providing the no-alpine image for arm64/v8. It should be easy to accomplish.

**Comment by BlackDex** [comment]
Unfortunately it isn't that easy.
Also, we probably could only add sqlite for this at the moment.
I am working on trying to solve this, but again, not easy because of some libraries which link to some C libraries and cause issues.

**Reply by ngosang** [reply]
Thank you! If you need help with testing let me know.
